cassandra-four-zero/src/main/java/org/apache/cassandra/spark/reader/SchemaBuilder.java

```diff
@@ -519,7 +519,7 @@ private List<CqlField> buildFields(TableMetadata metadata, Map<String, CqlField.
             boolean isPartitionKey = col.isPartitionKey();
             boolean isClusteringColumn = col.isClusteringColumn();
             boolean isStatic = col.isStatic();
-            String name = col.name.toCQLString();
+            String name = col.name.toString();
```

Comment:
`toCQLString` gives us the quoted string, but we immediately drop the quotes when creating the field. So we should just return the unquoted string, which is what we use